packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -95,6 +95,15 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
     }
   }
 
+  // Check if the input and output file paths are identical, otherwise return an
+  // error to the user.
+  if (args['--input'] === args['--output']) {
+    eprintln(header())
+    eprintln()
+    eprintln(`Specified input file ${highlight(relative(args['--input']))} and output file ${highlight(relative(args['--output']))} are identical.`)
+    process.exit(1)
+  }
```

Comment:
Thanks for the contribution! Do you mind adding a test for it as well? Check out the `integrations/cli` folder for some examples 👍 

Also we might consider doing a `path.resolve(…)` on the input and output first so that it will warn even in these cases:

```
tailwindcss --input foo.js --output ./node_modules/../foo.js
```

Reply:
Didn't find any tests - will add them in.

On the second part, this runs after the `path.resolve()` evaluation so would already handle this case.

Reply:
Oh, yep you're right, great!

Reply:
Added, and also added a test for input file not existing as that wasn't covered.